.github/workflows/derive-artefact-address.yaml

```diff
@@ -0,0 +1,43 @@
+name: Derive artefact address workflow
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        description: "The git tag to derive artefact location from"
+        required: true
+        type: string
+      workflow_timeout:
```

Comment:
is there a reason why we opted to pass this. I know for some jobs (eg spinning up cloudfront etc) we wanted to be able to extend the github default but this one looks pretty self-contained. Happy to keep it if we need it but if we don;t maybe its cleaner if we don't specify it as an optional input

Reply:
Removed timeout